src/components/FullscreenModal/hooks/useStickyFooter.ts

```diff
@@ -16,12 +18,21 @@ export const useStickyFooter = (
       modalRef.current.scrollHeight > modalRef.current.offsetHeight;
     setShouldShowScrollToTopButton(isScrollable);
     setIsFixed(false);
-  }, [modalRef, setShouldShowScrollToTopButton]);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
 
   useEffect(() => {
     showScrollToTopButton();
     window.addEventListener('resize', debounce(showScrollToTopButton, 200));
-  }, [modalRef, showScrollToTopButton]);
+
+    return () => {
+      window.removeEventListener(
+        'resize',
+        debounce(showScrollToTopButton, 200),
```

Comment:
i think `removeEventListener` removes by instance,  and `debounce` returns new function each time 🤔 
E.g. should be something like this:
```
const listener = debounce(showScrollToTopButton, 200);
window.addEventListener('resize', listener);
 return () => {
      window.removeEventListener(
        'resize',
        listener)
```